### DISCUSSION TITLE
How to include a source term in the continuity equation (INSFVMassAdvection)

### DISCUSSION THREAD

**Discussion by AnthonyB08** [idaholab/moose#23529] [Q&A Modules: Navier-Stokes] [answered]
Hi all,

I am trying to impose a arbitrary source term by utilizing the fvkernal : FVBodyForce.  The variable that this object residual is dependent on is pressure and the function is rho*2, i.e 2000. However, when I postprocess the simulation results with VolumetricFlowRate object  : 
 [inlet_mass]
    type = VolumetricFlowRate
    boundary ='left'
    vel_x = u
    vel_y = v
    advected_quantity = ${rho}
  []

I see no accumulation in my results: 
![image](https://user-images.githubusercontent.com/55299169/221225773-19ea8485-9da4-4a88-a0d7-3aa9c01735b8.png)

What am I not understanding about FVBodyForce or am I using the wrong postprocessing method? 

-Thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

In which equation did you add this source term?
(eg what variable did you specify for the kernel)

Guillaume

**Reply by AnthonyB08** [reply]
################## Source Term #########################
  [mass_source]
    type = FVBodyForce
    variable = pressure
    function = 2000
    #value=1000
  []
[]

**Reply by AnthonyB08** [reply]
This is how the source term is being implemented

**Reply by AnthonyB08** [reply]
I got it to work, disregard.
